bot/utils/scheduling.py

```diff
@@ -157,18 +157,18 @@ def _task_done_callback(self, task_id: t.Hashable, done_task: asyncio.Task) -> N
                 self._log.error(f"Error in task #{task_id} {id(done_task)}!", exc_info=exception)
 
 
-def create_task(*args, **kwargs) -> asyncio.Task:
+def create_task(coro: t.Awaitable, *suppressed_exceptions: Exception, **kwargs) -> asyncio.Task:
     """Wrapper for `asyncio.create_task` which logs exceptions raised in the task."""
-    task = asyncio.create_task(*args, **kwargs)
-    task.add_done_callback(_log_task_exception)
+    task = asyncio.create_task(coro, **kwargs)
+    task.add_done_callback(partial(_log_task_exception, *suppressed_exceptions))
```

Comment:
Does this actually work? I'm thinking it might put an exception in place of the `task` argument.

Reply:
In what case would that happen?

Reply:
Oh I see it now

Reply:
If it's a problem then I _think_ the fix would be to pass it as a tuple through a kwarg rather than unpacking it.

Reply:
I'll just make it take a `tuple` since it's only going to be used once.